SimPEG/regularization/pgi.py

```diff
@@ -767,26 +1233,94 @@ def alpha_pgi(self, value):
 
     @property
     def gmm(self):
+        """Gaussian mixture model.
+
+        Returns
+        -------
+        None, SimPEG.utils.WeightedGaussianMixture
+            Gaussian mixture model. If ``None``, the Gaussian mixture model is learned throughout the
+            inversion. If set, the Gaussian mixture model used to constrain the recovered model is
+            static throughout the inversion.
+        """
         return self.objfcts[0].gmm
 
     @gmm.setter
     def gmm(self, gm):
         self.objfcts[0].gmm = copy.deepcopy(gm)
 
     def membership(self, m):
+        """Compute and return membership array for the model provided.
+
+        The membership array stores the index of the rock unit most representative of each cell.
+        For a Gaussian mixture model containing the means and covariances for model parameter
+        types (physical property types) for all rock units, this method computes the membership
+        array for the model `m` provided. For a description of the membership array, see the
```

Comment:
```suggestion
        array for the model ``m`` provided. For a description of the membership array, see the
```

Reply:
In this case, m is an input argument, which we are usually denoting as `m`. In this case it's not a property, so no ``m``. Agree?

Reply:
Sure! No big deal.